README.md

```diff
@@ -1,13 +1,14 @@
 # funding-service-design-utils
-Shared library for funding service design apps.
+Utils serves as a shared library for funding service design applications.These utilities are meant to be reused across the project, streamlining our coding process and ensuring the reliability of our funding service design applications.
 
 This library can be installed into other python repos and the packages used by those repos.
 
+This service depends on:
+- No other microservices
+
 # Dev setup
 In order to run the unit tests, setup a virtual env and install requirements
-1. Checkout the code
-1. Setup a venv and activate: `python3 -m venv .venv && source .venv/bin/activate`
-1. Install dev requirements: `pip install --upgrade pip && pip install -r requirements-dev.txt`
+1. [Developer setup guide](https://github.com/communitiesuk/funding-service-design-workflows/blob/main/readmes/python-repos-setup.md)
 1. Install pre-commit hook: `pre-commit install`
```

Comment:
These pre-commit and testing bits are also covered in the shared workflows, so please link to them :)